Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -12274,9 +12314,7 @@ - (void)contextMenuInteraction:(UIContextMenuInteraction *)interaction willEndFo
     _contextMenuHasRequestedLegacyData = NO;
     _contextMenuElementInfo = nullptr;
 
-#if ENABLE(IMAGE_ANALYSIS_FOR_MACHINE_READABLE_CODES)
-    _contextMenuForMachineReadableCode.clear();
-#endif // ENABLE(IMAGE_ANALYSIS_FOR_MACHINE_READABLE_CODES)
+    _imageAnalysisContextMenuActionData = std::nullopt;
```

Comment:
This used to just clear the MRC menu, but will now also remove the "copy subject" image.  Is that OK?

Reply:
That's an excellent catch — while I had confirmed locally that "Copy Subject" still works, I'll do some more investigation to confirm that we can rely on this always being cleared out after we've already invoked the action handler.

Reply:
So given UIKit's current implementation, it's guaranteed that the action is going to be invoked before the `-willEndForConfiguration:`. However, it does seem slightly fragile to rely on this ordering — I think a more robust way to deal with this state reset is to actually lazily reset it when we're about to compute `_imageAnalysisContextMenuActionData`, inside `-_completeImageAnalysisRequestForContextMenu:` (so that way we never end up using stale results).